internal/controllers/machine/machine_controller_noderef.go

```diff
@@ -46,7 +46,7 @@ func (r *Reconciler) reconcileNode(ctx context.Context, cluster *clusterv1.Clust
 
 	// Check that the Machine has a valid ProviderID.
 	if machine.Spec.ProviderID == nil || *machine.Spec.ProviderID == "" {
-		log.Info("Cannot reconcile Machine's Node, no valid ProviderID yet")
+		log.Info("Waiting for infrastructure provider to report spec.ProviderID, requeing", util.LowerCamelCase(machine.Spec.InfrastructureRef.Kind), klog.KRef(machine.Spec.InfrastructureRef.Namespace, machine.Spec.InfrastructureRef.Name))
```

Comment:
It's a low level detail, but should we use requeueing when we're not doing that in the controller runtime sense?

Reply:
Removed